packages/core/src/common/selection-service.ts

```diff
@@ -15,22 +16,30 @@ export interface SelectionProvider<T> {
 }
 
 @injectable()
-export class SelectionService implements SelectionProvider<any> {
+export class SelectionService implements SelectionProvider<Object | undefined> {
```

Comment:
why not any? Having a string as the selection would make sense, no?

Reply:
with `any` there is no way to distinguish between `undefined` and value, with `Object | undefined` they are forced to check for `undefined` and add a proper type guard

Reply:
`Object` allows `string`, `object` does not allow